tests/baselines/reference/sourceMap-LineBreaks.js

```diff
@@ -3,14 +3,17 @@
 //// [sourceMap-LineBreaks.ts]
 var endsWithlineSeparator = 10;  var endsWithParagraphSeparator = 10;  var endsWithNextLine = 1;var endsWithLineFeed = 1;
 var endsWithCarriageReturnLineFeed = 1;
-var endsWithCarriageReturn = 1;
var endsWithLineFeedCarriageReturn = 1;
-
var endsWithLineFeedCarriageReturnLineFeed = 1;
+var endsWithCarriageReturn = 1;
+var endsWithLineFeedCarriageReturn = 1;
+
+var endsWithLineFeedCarriageReturnLineFeed = 1;
```

Comment:
I am trying so hard to get this test back the way it was, but every editor is fighting me

Reply:
Finally, turns out it's git's fault and I had to force squash and then fix in order to not have any intermediary diffs